contrib/munin/minion

```diff
@@ -52,6 +57,8 @@ sub main {
 sub data {
     my ($config, $category) = @_;
     my $url = $ENV{url} || "https://openqa.opensuse.org/admin/influxdb/minion";
+    my $span = 5; # not configurable (yet)
```

Comment:
Why did you not make this configurable yet? I guess it isn't a necessity but the comment makes one wonder why.

Reply:
Because I haven't found out if it's possible to adjust the timespan on munin side.
And it would only make sense if we can do that.
The only other way I could think is that we divide the value, e.g. if we configure a timespan of 20 minutes, we would divide the value y 4 to display correct values.